### DISCUSSION TITLE
[@types/encoding-japanese] convert function data: type missing

### DISCUSSION THREAD

**Discussion by RibikiH** [DefinitelyTyped/DefinitelyTyped#60318] [Issues with a @types package]
Please add data type: `number[]` to convert function
`export function convert(data: string | number[], to: Encoding, from?: Encoding): string;`
 @rhysd

[a comment by an automated account is not shown]

**Reply by rhysd** [reply]
Can you make a PR?

**Reply by RibikiH** [reply]
yes, i created https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60321
